### ISSUE TITLE
REACT QUERY IS SLOWING DOWN EXPO APP

### ISSUE THREAD

**Issue by oxneat** [tannerlinsley/react-query#1497]
I just added react query to my expo app and after playing with a little bit i noticed a huge drop in performance the app isn't interactive anymore i would press button a and it would not trigger right away only after several seconds , the navigation is also slow and laggy

**Comment by tannerlinsley** [comment]
I'm sorry you're experiencing this, but there's not really any way to be sure if this is React Query's fault though because you haven't supplied us with anything to start debugging.

- Please do not file issues with all caps titles in the future.
- A minimal reproduction would go a long way, and if it's one we can just open and debug in the browser or through the expo app, then that would be even better.
- You may be tempted to further explain how everything worked fine before adding React Query, but keep an open mind that adding new technology to a system can sometimes uncover unknown side-effects in our own code that we didn't expect.

Please comment with more info (and we'll reopen this issue) or feel free to file a new issue with that same info. Either way, we'll appreciate your time in helping us figure it out :)
